src/main/java/software/aws/neptune/jdbc/utilities/ConnectionProperties.java

```diff
@@ -40,24 +41,26 @@
  * Class that contains connection properties.
  */
 public abstract class ConnectionProperties extends Properties {
-    public static final String SCAN_TYPE_KEY = "ScanType";
-    public static final String APPLICATION_NAME_KEY = "ApplicationName";
-    public static final String AUTH_SCHEME_KEY = "AuthScheme";
-    public static final String CONNECTION_TIMEOUT_MILLIS_KEY = "ConnectionTimeout";
-    public static final String CONNECTION_RETRY_COUNT_KEY = "ConnectionRetryCount";
-    public static final String LOG_LEVEL_KEY = "LogLevel";
+    public static final String SCAN_TYPE_KEY = "scanType";
+    public static final String APPLICATION_NAME_KEY = "applicationName";
+    public static final String AUTH_SCHEME_KEY = "authScheme";
+    public static final String CONNECTION_TIMEOUT_MILLIS_KEY = "connectionTimeout";
+    public static final String CONNECTION_RETRY_COUNT_KEY = "connectionRetryCount";
+    public static final String LOG_LEVEL_KEY = "logLevel";
     public static final String SSH_USER = "sshUser";
     public static final String SSH_HOSTNAME = "sshHost";
     public static final String SSH_PRIVATE_KEY_FILE = "sshPrivateKeyFile";
     public static final String SSH_PRIVATE_KEY_PASSPHRASE = "sshPrivateKeyPassphrase";
     public static final String SSH_STRICT_HOST_KEY_CHECKING = "sshStrictHostKeyChecking";
     public static final String SSH_KNOWN_HOSTS_FILE = "sshKnownHostsFile";
+    public static final String SERVICE_REGION_KEY = "serviceRegion";
```

Comment:
We should ensure documentation parameter is also added in the doc.

Reply:
Noted.